test/libyul/yulInterpreterTests/access_large_memory_offsets.yul

```diff
@@ -7,8 +7,6 @@
 // ----
 // Trace:
 // Memory dump:
-//      0: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff07
-//   FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE0: 00000000000000000000000000000000000000000000000000000000000000ff
+//      0: 0000000000000000000000000000000000000000000000000000000000000007
 // Storage dump:
 //   0000000000000000000000000000000000000000000000000000000000000000: 0000000000000000000000000000000000000000000000000000000000000007
-//   0000000000000000000000000000000000000000000000000000000000000001: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```

Comment:
If this change removes this write, won't this cause *more* false positives?
For the optimizer it's just fine to resolve the ``mload(sub(0,1))`` to ``sub(0,1)`` - which would result in ``sub(0,1)`` be written to storage slot ``1``...

Reply:
You are right, this may cause more false positives because loads from previously written to (large) memory addresses won't be interpreted.

I don't have a good solution to the problem this PR seeks to address.